data/json/recipes/practice/melee.json

```diff
@@ -0,0 +1,157 @@
+[
+  {
+    "id": "prac_melee_beg",
+    "type": "practice",
+    "activity_level": "MODERATE_EXERCISE",
+    "category": "CC_PRACTICE",
+    "subcategory": "CSC_PRACTICE_COMBAT",
+    "name": "melee (beginner)",
+    "description": "Practice close quarters combat with and without weapons against an imaginary opponent.",
+    "skill_used": "melee",
+    "time": "1 h",
+    "practice_data": { "min_difficulty": 0, "max_difficulty": 2, "skill_limit": 3 },
+    "autolearn": [ [ "melee", 1 ] ],
+    "book_learn": [ [ "mag_melee", 0 ], [ "manual_melee", 0 ] ],
+    "//": "Only training weapons, all of these have the WBLOCK_2 techniques and are relatively easy to handle, in other words they have a possitive to_hit bonus.",
+    "tools": [ [ [ "cudgel", -1 ], [ "bokken", -1 ], [ "bokken_fake", -1 ], [ "bokken_inferior", -1 ], [ "q_staff", -1 ] ] ]
```

Comment:
You don't need the -1, that's just for tools with charges.
Instead of `[ "foo", -1 ]` just write `"foo"` - it's simpler.
So this line becomes
```
    "tools": [ [ "cudgel", "bokken", "bokken_fake", "bokken_inferior", "q_staff" ] ]
```

Reply:
Working on it!